### ISSUE TITLE
For Cycle 12

### ISSUE THREAD

**Issue by pedromvpg** [bisq-network/compensation#520]
## Summary

 - BSQ requested: 1800 BSQ * 0.44 = 792 USD

## Contributions delivered

Design and customized the CSS for the new bisq wiki (https://bisq.wiki/)

![bisq_wiki_before_after](https://user-images.githubusercontent.com/807505/77990478-7857ed80-72ef-11ea-8e64-d10393a718b7.png)

**Comment by MwithM** [comment]
There's been a [change ](https://github.com/bisq-network/compensation/issues/519)in the way USD per BSQ is calculated. Now, for this cycle, it's 0.56 USD/BSQ.

Please update.

**Comment by m52go** [comment]
The wiki is mainly a support resource—so I think it would be better to assign @cbeams to sign off here.

**Comment by cbeams** [comment]
@pedromvpg, assuming you're working backwards from 792 USD (i.e. what you would have requested for this work in a normal freelancing situation), then the total BSQ at the new rate that @MwithM mentioned would be 792 / 0.56 = 1,414 BSQ. Please confirm you're OK with that and update the description of this issue to reflect. Thanks!

**Comment by cbeams** [comment]
@MwithM, not sure this should have been moved to `Review Complete`, since @pedromvpg didn't respond to my comment above. I pinged him in Keybase as well.

In any case, though, I see @pedromvpg has already submitted his DAO proposal with txid 5c3438910dec9c49166a36d5cab94f2646f64a6841a9d9f908f0e18860d1f5cb for 1800 BSQ, so I would prefer to just approve this proposal as-is and let it go through and deal with the discrepancy in the next compensation request.

So with that said, 
my [team lead review](https://bisq.wiki/Compensation#Team_Lead) is complete and I approve this request.
